src/backend/distributed/commands/table.c

```diff
@@ -309,11 +310,12 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand)
 	char leftRelationKind = get_rel_relkind(leftRelationId);
 	if (leftRelationKind == RELKIND_INDEX)
 	{
-		leftRelationId = IndexGetRelation(leftRelationId, false);
+		bool missingOk = false;
+		leftRelationId = IndexGetRelation(leftRelationId, missingOk);
 	}
 
-	bool isDistributedRelation = IsDistributedTable(leftRelationId);
-	if (!isDistributedRelation)
+	bool referencingIsLocalTable = !IsDistributedTable(leftRelationId);
+	if (referencingIsLocalTable)
```

Comment:
I suppose that this is currently true, though I feel like the previous definition was a bit more correct since `IsDistributedTable` does not say much about the "localness" of the table. e.g. it could be a foreign table.

Could also be `if (!IsDistributedTable(leftRelationId))`

Reply:
Actually I wanted to make it more clear as I will/am use/using those variables while implementing my fkey between _local tables_ & reference tables task